### PR TITLE
fix collborators lack of units on orgnization repositories

### LINKED ISSUE #1947: Unable to git-push to existing repos (probably related to the unit feature)
- Gitea version (or commit ref):
Gitea Version: 1.1.0+292-g113b3e9

- Git version:
2.7.4

- Operating system:
Ubuntu 16.04 64bit

- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite

- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
Got rejected and an error message `User Scott does not have allowed access to repository my/test-repo's code` while doing `git push`. It worked before.

A quick search gives me the clue but I don't quite understand what it's about.
````
if !repo.CheckUnitUser(authUser.ID, authUser.IsAdmin, unitType) {
    ctx.HandleText(http.StatusForbidden, fmt.Sprintf("User %s does not have allowed access to repository %s 's code",
    authUser.Name, repo.RepoPath()))
    return
}
```
...

### PR DESCRIPTION
**Pull request by lunny**
will fix #1947, @typeless please confirm this PR fixed #1947

### CODE REVIEW

File: integrations/sqlite.ini [hunk 1]
```diff
@@ -46,10 +46,10 @@ PROVIDER = file
 MODE = console,file
 
 [log.console]
-LEVEL = Warn
+LEVEL = Debug
```
Comment (ethantkoenig) [inline integrations/sqlite.ini:49]: Why change `sqlite.ini` but not `mysql.ini` or `pgsql.ini`?

**Comment by ethantkoenig** [comment]
One minor thing, otherwise LGTM

File: integrations/sqlite.ini (hunk 1 above)
Comment (lunny) [reply integrations/sqlite.ini:49]: done.

**Comment by typeless** [comment]
It works.

**Comment by bkcsoft** [comment]
LGTM

**Pull request merged**